pyproject.toml

```diff
@@ -27,7 +27,7 @@ dependencies = [
     "newscale@git+https://github.com/AllenNeuralDynamics/python-newscale@axes-on-target",
     "aind-auto-train@git+https://github.com/AllenNeuralDynamics/aind-foraging-behavior-bonsai-automatic-training.git@main",
     "aind-slims-api@git+https://github.com/AllenNeuralDynamics/aind-slims-api@main",
-    "aind-dynamic-foraging-models@git+https://github.com/AllenNeuralDynamics/aind-dynamic-foraging-models@main",
+    "aind-dynamic-foraging-models@git+https://github.com/AllenNeuralDynamics/aind-dynamic-foraging-models@v0.11.2",
     "aind-behavior-services<0.9",
```

Comment:
the package is available on pypi, so I think its better to do `aind-dynamic-foraging-models>=0.11.2`, so we aren't pinned to that exact release